releasenotes/notes/auto_instrumentation_disable_dotnet_profiler-64772048cafb07b7.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This file should be in the release notes of the Cluster agent instead of the core agent: `releasenotes-dca/notes/`